app/views/audits/index.html.erb

```diff
@@ -15,18 +15,17 @@
     </tr>
   </thead>
   <tbody class="govuk-table__body">
-    <tr class="govuk-table__row">
-      <td class="govuk-table__cell"> <%= @planning_application.created_at.strftime("%d-%m-%Y %H:%M") %></td>
-      <td class="govuk-table__cell"> API </td>
-      <td class="govuk-table__cell"> Application created </td>
-    </tr>
     <% @audits.each do |audit_item| %>
       <tr class="govuk-table__row">
         <td class="govuk-table__cell"><%= audit_item.created_at.strftime("%d-%m-%Y %H:%M") %></td>
-        <td class="govuk-table__cell"><%= audit_item.user.name %></td>
+        <% if audit_item.user %>
+          <td class="govuk-table__cell"><%= audit_item.user.name %></td>
+        <% elsif audit_item.api_user %>
+          <td class="govuk-table__cell"><%= audit_item.api_user.name %></td>
+        <% end %>
         <td colspan="2" class="govuk-table__cell">
-          <%= activity(audit_item.activity_type, audit_item.activity_information) %> <br>
-          <i> <%= audit_item.audit_comment if audit_item.audit_comment.present? %> </i>
+          <%= activity(audit_item.activity_type, audit_item.activity_information) %>
+          <i> <%= simple_format(audit_item.audit_comment, {}, wrapper_tag: "div") if audit_item.audit_comment.present? %> </i>
```

Comment:
Two things here: I think the i tag is frowned upon, and em is preferred? I might be out of date here though. But also what you end up doing here is wrapping a p tag (created by the simple_format) in an i tag, when it should be the other way around. I can't easily find a way of specifying a class name on the p tag simple_format creates, so maybe the best solution is the wrap this all in a div with a class name and set the CSS on that div to italicize the text?

Reply:
I can add a class to audit_item to make it italicize with the gov uk library if we don't like i tags (although I don't see how they are different from other tags (?) 

doing ```wrapper_tag div ``` ensures we get the content inside a div instead of inside a paragraph, because the paragraph was adding extra line breaks in between content which we didn't want. so just 

```
simple_format(audit_item.audit_comment

```
produced this:

![image](https://user-images.githubusercontent.com/9452321/111156939-31114e00-858e-11eb-9269-ffa89db7902a.png)

but what we currently have produces this:

![image](https://user-images.githubusercontent.com/9452321/111156977-3cfd1000-858e-11eb-9c0a-0c383fbb9b5e.png)
![image](https://user-images.githubusercontent.com/9452321/111157156-76ce1680-858e-11eb-914f-0902033bddeb.png)

Reply:
I was suggesting manually wrapping the simple_format in a div instead of `i`, so:

`<div class="audit_details"><%= simple_format(audit_item.audit_comment, {}, wrapper_tag: "div") if audit_item.audit_comment.present? %></div>`

and then having

```
.audit_details {
  font-style: italic;
}
```

in the CSS

Reply:
context: `i` used to mean `italic`, which is presentational not semantic, so is frowned upon, with a preference for `em`. I see that they've changed the meaning of `i` to mean "[Idiomatic Text](Idiomatic Text)", which is news to me, but I see govuk still suggests `em`, so I'd stick with that.

But in this context, either `i` or `em` is an inline element, and `p` is a block element, and you shouldn't wrap a block element in an inline element.

Reply:
that makes sense thanks - I'll change it